docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: nexus
+title: Temporal Nexus - Temporal feature
+description: Explore how Temporal Nexus enables modular architecture and secure multi-team collaboration by abstracting Temporal primitives into clean Nexus Service APIs across namespace boundaries.
+sidebar_label: Temporal Nexus
+slug: /temporal-nexus
+tags:
+  - temporal-nexus
+  - temporal-cloud
+
+keywords:
+  - temporal multi-team collaboration
+  - modular temporal architecture
+  - namespace boundary workflows
+  - temporal nexus api registry
+  - temporal nexus services
+  - secure temporal workflows
+  - multi-team integration temporal
+  - temporal cloud nexus
+---
+
+:::tip Support, stability, and dependency info
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for both OSS and Temporal Cloud.
```

Comment:
think we can omit this here as there are not many places in evaluate that mention OSS/open source since it's implicit as most features are in the open source, of note:
- major-components.mdx
- temporal-cloud/overview.mdx - mentions at the bottom of the overview page
- release-stages

self-host has more along with some in references or encyclopedia

```suggestion
```

Reply:
This also came out the convo with @irinabelova12. I do agree that it can be implied and otherwise, we'll have to make that differentiation for all of the pages which might not be neccessary at the moment.